src/shared/constants.ts

```diff
@@ -27,3 +27,13 @@ export const FORM_DUPLICATE_KEYS = [
   'authType',
   'inactiveMessage',
 ]
+
+// Config for AWS.
+export const AWS_DEFAULT = {
+  region: 'ap-southeast-1',
+  endpoint: 'http://localhost:4572',
```

Comment:
Hm, tbh I'm not sure if this belongs here. The endpoint is something which is different between dev and prod environments, which is the definition of an env var. So should `AWS_ENDPOINT` be an env var as well? This would also solve our validation problem - you can validate `AWS_ENDPOINT` just like all the s3 bucket URLs.

The only drawback I see to this is that you'd have to hardcode the AWS region in the endpoint, instead of letting it be a variable. But I don't think we're going to change region anytime soon (or ever).

Reply:
i like the idea. but it opens up the possibility of production being configured wrongly? The region being set as `a`, and the `S3` sdk being configured as such but the `awsEndpoint` itself has a different region?

Reply:
@arshadali172

Reply:
agreed, but how likely is that to happen given that we've never changed our AWS region? personally I'd rather have the code be cleaner and have a customised env var. I don't think it's very important to leave the endpoint as a variable which depends on the region. if one day we do change AWS regions, I would think we'd be careful enough to review all our env vars and test on staging before deploying the change.